### PR TITLE
[qa] Format Lua Files #51

### LINKED ISSUE #51: [qa] Format lua files
Format lua files to be consistent with openwisp-config in general and following lua best practices

- [x] Whitespace around operators
- [x] Search for a lua autolinter
- [x] Add documentation about it in README

### PR DESCRIPTION
**Pull request by devkapilbansal**
Closes #51

### CODE REVIEW

**Comment by devkapilbansal** [comment]
Converting to draft as documentation is left

File: README.rst
Comment (nemesisdesign) [inline README.rst:143]: are you sure this second part is needed at all? The syntax may change in the future and we may forget to update this here, will people working on this module really need to know this?

**Review by nemesisdesign — COMMENTED** [review]
This is great, much needed!
We should wait for the other PRs to get merged before merging this one because it will generate painful conflicts.

**Comment by devkapilbansal** [comment]
Tests are failing due to a minor error in `network_data` introduced while resolving merge conflicts. Will update once #34 and #48 are merged

File: openwrt-openwisp-monitoring/files/lib/openwisp/dhcp.lua
Comment (nemesisdesign) [inline openwrt-openwisp-monitoring/files/lib/openwisp/dhcp.lua:15]: this line is quite long, is there a way to set a max length so that long lines are splited on multiple lines?

File: README.rst
Comment (nemesisdesign) [reply README.rst:143]: is still here though

File: openwrt-openwisp-monitoring/luaformat.config
Comment (devkapilbansal) [inline openwrt-openwisp-monitoring/luaformat.config:4]: This file can help define the config.

File: openwrt-openwisp-monitoring/files/lib/openwisp/dhcp.lua
Comment (devkapilbansal) [reply openwrt-openwisp-monitoring/files/lib/openwisp/dhcp.lua:15]: What should be the max_length? Luacheck checks if line is greater than 120 characters or not. Hence, I followed its convention and set the max_length to 120

File: README.rst
Comment (devkapilbansal) [reply README.rst]: :grimacing: Remvoed again

File: openwrt-openwisp-monitoring/files/lib/openwisp/dhcp.lua
Comment (nickberry17) [reply openwrt-openwisp-monitoring/files/lib/openwisp/dhcp.lua]: 80 characters is the traditional limit

**Review by nemesisdesign — APPROVED** [review]
Thanks!

**Pull request merged**
